### PR TITLE
Updates to README.md

### PR DESCRIPTION
**Pull request by banciur**

### CODE REVIEW

**Review by Moejoe90 — APPROVED** [review]

**Review by Moejoe90 — APPROVED** [review]
I just emphasize on Airbnb rules to be set as an option if its not a default

File: README.md [hunk 1]
```diff
@@ -41,3 +49,24 @@ C: `yarn test-watch`
 D: (use for issuing random commands like `git`)
 
 Open Chrome and go to [https://localhost:8080/](https://localhost:8080/).
+
+### Using starter kit as starting point for new projects
+You need to create branch that tracks this repository and rebase it on your master. Then when there will be never versions of starter kit (new features, bugfixes etc) you can pull those using starter branch and merge to your project.
```
Comment (recmo) [inline README.md]: `never` → `newer`

File: README.md [hunk 2]
```diff
@@ -41,3 +49,24 @@ C: `yarn test-watch`
...
+
+Lets assume you created ```new_project``` use following commands to setup it:
```
Comment (recmo) [inline README.md]: `use following` → `use the following`

File: README.md [hunk 3]
```diff
@@ -41,3 +49,24 @@ C: `yarn test-watch`
...
+    git commit --allow-empty -m "Initial commit"
+    git remote add dapp-starter-kit git@github.com:Neufund/dapp-starter-kit.git
+    git fetch dapp-starter-kit
+    git branch starter
+    git branch --set-upstream-to=dapp-starter-kit/master starter
+    git checkout starter
+    git pull -r
+    git checkout master
+     
+ignore msg "Your branch is based on 'origin/master', but the upstream is gone." - it's beacuse your master is empty thanks to usage of "--allow-empty" parameter
```
Comment (recmo) [inline README.md]: `Ignore the message`

**Review by recmo — CHANGES_REQUESTED** [review]
Great additions! There are some minor typos.

**Review by recmo — APPROVED** [review]

**Pull request merged**